openssl-sys/src/lib.rs

```diff
@@ -345,25 +380,31 @@ extern "C" {
     pub fn EVP_aes_128_ecb() -> *const EVP_CIPHER;
     #[cfg(feature = "aes_xts")]
     pub fn EVP_aes_128_xts() -> *const EVP_CIPHER;
-    // fn EVP_aes_128_ctr() -> EVP_CIPHER;
-    // fn EVP_aes_128_gcm() -> EVP_CIPHER;
+    pub fn EVP_aes_128_ctr() -> *const EVP_CIPHER;
```

Comment:
Should each feature be gated individually or would it make sense for them to be branch based or even just "modern_openssl" for stuff not present in OpenSSL 0.x? There isn't a build-time feature detection phase anyway, is there?
Speaking of versions, would you mind putting an official statement in the Readme as to which OpenSSL versions or branches the bindings are expected to support?